packages/button/src/__stories__/stories.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import Icon, { names as iconNames } from "@hig/icon";
+
+export default [
+  {
+    description: "default",
+    getProps: () => ({
+      title: "Button Title"
+    })
+  },
+  {
+    description: "as a link",
+    getProps: () => ({
+      link: "https://www.autodesk.com",
```

Comment:
This example as `onClick` in storybook, which is not likely to be used in conjunction with `link`.

Reply:
Possibly. There's always analytics click tracking 🤷‍♂️ .